site/search/Autocomplete.tsx

```diff
@@ -243,7 +244,8 @@ const AlgoliaSource: AutocompleteSource<BaseItem> = {
 }
 
 const createFiltersSource = (
-    allTopics: string[]
+    allTopics: string[],
+    synonymMap: Map<string, string[]>
```

Comment:
Maybe make a `SynonymMap` type? Then you could lock down the 7 instances of `Map<string, string[]>` in this PR